test/Quake/add_measurements-1.qke

```diff
@@ -0,0 +1,102 @@
+// ========================================================================== //
+// Copyright (c) 2025 NVIDIA Corporation & Affiliates.                        //
+// All rights reserved.                                                       //
+//                                                                            //
+// This source code and the accompanying materials are made available under   //
+// the terms of the Apache License 2.0 which accompanies this distribution.   //
+// ========================================================================== //
+
+// RUN: cudaq-opt --add-measurements %s | FileCheck %s
+
+module attributes {quake.mangled_name_map = {__nvqpp__mlirgen__caller = "__nvqpp__mlirgen__caller_PyKernelEntryPointRewrite"}} {
+  func.func @__nvqpp__mlirgen__subkernel() attributes {"cudaq-entrypoint", "cudaq-kernel"} {
+    %0 = quake.alloca !quake.ref
+    quake.x %0 : (!quake.ref) -> ()
+    %measOut = quake.mz %0 : (!quake.ref) -> !quake.measure
+    return
+  }
```

Comment:
Probably should have a test with multiple `return` Ops. `return %5 : T` would be a good idea as well.